docs/1.3-supported-os.md

```diff
@@ -0,0 +1,40 @@
+# Composer 1.3 - Recommended OS versions
+
+The tables below provide recommended OS information for Composer 1.3.
+
+For this release, Composer will **only run on x64**. We will support more architectures in upcoming releases.
+
+## Windows
+
+OS                | Version            | Architectures |
+------------------|--------------------|---------------|
+Windows 10        | Version 1607+      | x64           |
+Windows Server    | 2012 R2+           | x64           |
+Nano Server       | Version 1803+      | x64           |
+Windows Client    | 7 (SP1), 8.1       | x64           |
```

Comment:
I'm not sure we should recommend windows 7 or 8.1

Reply:
sounds good, will take them out.